j/jbuilder/jbuilder_ubi8.4.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package	: jbuilder
+# Version	: v2.11.5
+# Source repo	: https://github.com/rails/jbuilder
+# Tested on	: UBI 8.4
+# Language      : Ruby
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Sapana Khemkar {Sapana.khemkar@ibm.com}
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=jbuilder
+PACKAGE_VERSION=${1:-v2.11.5}
+PACKAGE_URL=https://github.com/rails/jbuilder.git
+
+yum install -y git wget curl ruby ruby-devel rubygem-rake procps libcurl-devel libffi-devel #sqlite sqlite-devel nodejs nodejs-devel nodejs-packaging 
+
+yum install -y --allowerasing gcc gcc-c++ yum-utils make automake autoconf libtool gdb* binutils rpm-build gettext 
+
+yum-config-manager --add-repo http://mirror.centos.org/centos/8/AppStream/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/PowerTools/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/BaseOS/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/virt/ppc64le/ovirt-44/
```

Comment:
I am not sure this will work now and in any case we need to remove unwanted repos and packages